tests/token.test.ts

```diff
@@ -209,5 +238,44 @@ describe.each([{ permission: 'Private' }])(
         searchClient.index(UID).search('pride')
       ).rejects.toHaveProperty('code', 'invalid_api_key')
     })
+
+    test(`${permission} key: Search in tenant token with expired token`, async () => {
+      const client = await getClient(permission)
+      const apiKey = await getKey(permission)
+      const { uid } = await client.getKey(apiKey)
+      const date = new Date('December 17, 2000 03:24:00')
+
+      expect(() =>
+        client.generateTenantToken(
+          uid,
+          {},
+          {
+            expiresAt: date,
+          }
+        )
+      ).toThrowError(
+        new MeiliSearchError(`Meilisearch: The token has expired.\n`)
+      )
+    })
+
+    test(`${permission} key: Search in tenant token with wrong uid type`, async () => {
```

Comment:
to me is more like: `creates tenant token with an uid without a valid UUID v4 format throws an error`